src/twisted/test/test_ssl.py

```diff
@@ -667,8 +671,10 @@ def test_method(self):
         # SSLv23_METHOD allows SSLv2, SSLv3, or TLSv1
         self.assertEqual(self.context._method, SSL.SSLv23_METHOD)
 
-        # And OP_NO_SSLv2 disables the SSLv2 support.
-        self.assertTrue(self.context._options & SSL.OP_NO_SSLv2)
+        # OpenSSL >= 1.1.0 no longer provides SSLv2
+        if not OPENSSL_1_1:
```

Comment:
You can still test the flag when the value is 0! `assert (value & SSL_OP_NO_SSLv2) == SSL_OP_NO_SSLv2`